geopandas/geodataframe.py

```diff
@@ -664,6 +666,39 @@ def to_file(
 
         _to_file(self, filename, driver, schema, index, **kwargs)
 
+    def set_crs(self, crs=None, epsg=None, inplace=False, allow_override=False):
+        """
+        Set the Coordinate Reference System (CRS) of the ``GeoDataFrame``.
+
+        Set the CRS of the active geometry column of the GeoDataFrame, without
+        transforming the geometries. For actually transforming the geometries to
+        a new CRS, use the ``to_crs`` method.
+
+        Parameters
+        ----------
+        crs : pyproj.CRS, optional if `epsg` is specified
+            The value can be anything accepted
+            by :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`,
+            such as an authority string (eg "EPSG:4326") or a WKT string.
+        epsg : int, optional if `crs` is specified
+            EPSG code specifying the projection.
+        inplace : bool, default False
+            If set to True, the CRS of the GeoDataFrame will be changed in place
```

Comment:
> the CRS of the GeoDataFrame will be changed

This it technically not correct as we change the CRS of the active geometry column. Are we fine with the shortcut `CRS of the GeoDataFrame == CRS of the active geometry`? The same is below.

Reply:
Ah, yes, this is from before that change ;)

I think in general "CRS of the GeoDataFrame" meaning "CRS of the active gemetry column" is fine to say in the docs (it's in the end also what it is in practice `gdf.crs == gdf.geometry.crs`). But maybe it's good to be more explicit here (or add the second in brackets)

Reply:
The description block above should define this behavior with additional detail as necessary, since it holds true regardless of behavior of `inplace`.  

```
Set the Coordinate Reference System (CRS) of the ``GeoDataFrame`` and the active geometry column.

If there are multiple geometry columns within the GeoDataFrame, only the active geometry column is updated to this CRS.

NOTE: The underlying geometries are not transformed to this CRS.  To transform the geometries to a new CRS, use the ``to_crs`` method.
```

The notes for the `inplace` parameter can then be more succinct:

```
If True, the CRS will be updated in the existing GeoDataFrame.  If False, a new GeoDataFrame will be returned with this CRS.
```

Reply:
Thanks for the suggestion, updated!

For the inplace keyword, I kept the original though: the fact that with `inplace=True` the geodataframe is still returned is worth mentioning, I think, as that deviates from the typical pandas behaviour. And then to contrast it with `inplace=False`, I think it is important to mention "copy" (as both True and False return a GeoDataFrame). 
If this was not actually clear from the current explanation, and if you have suggestions to improve that, that's certainly welcome.